### ISSUE TITLE
File Versions dont open up when click them. 

### ISSUE THREAD

**Issue by AndyXheli** [nextcloud/richdocuments#2594]
**Describe the bug**

**To Reproduce**
1. Go to word file
2. Click on go into Versions 
3. Click on the file version nothing happens 

**Expected behavior**
Once the have has been selected Nextcloud office should open up that file version 

**Screenshots**

https://user-images.githubusercontent.com/59488153/199279066-f64980d0-eb6a-4939-aa02-1a5b0fe0db86.mp4

## Server details

**Operating system**:
Ubuntu 20.0.4
**Web server:**
Apache2
**Database:**
MySQL
**PHP version:**
8.0
**Nextcloud version:**
24.0.6
**Version of the richdocuments app**
6.3.1
**Version of Collabora Online**
22.05.7.2

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by juliushaertl** [comment]
Yes, version preview is currently only possible when the file is opened already in Collabora.

**Comment by AndyXheli** [comment]
Still an issue NC 27.0.1

**Comment by joshtrichards** [comment]
Fixed in #3171 / #3152 / #3172

**Comment by AndyXheli** [comment]
Thank you!!
